packages/compass-e2e-tests/helpers/commands/connect-with-connection-form.ts

```diff
@@ -1,16 +1,19 @@
 import type { CompassBrowser } from '../compass-browser';
 import type { ConnectFormState } from '../connect-form-state';
-import * as Selectors from '../selectors';
 
 export async function connectWithConnectionForm(
   browser: CompassBrowser,
   options: ConnectFormState,
   connectionStatus: 'success' | 'failure' | 'either' = 'success',
   timeout?: number
 ): Promise<void> {
-  const sidebar = await browser.$(Selectors.SidebarTitle);
-  if (await sidebar.isDisplayed()) {
-    await browser.disconnect();
+  await browser.disconnect();
```

Comment:
Just made this in line with other places where we disconnect. No need to check anything. We'd have to branch in the checking.

Reply:
I wonder if we could name the disconnect action differently then, something like ensureNoActiveConnections. sounds weird, but without your comment and checking the disconnect implementation I'd not think it works like it does.

Reply:
I renamed it to disconnectAll, but I'm generally reserving those kinds of refactoring/distinctions for when we add multiple connections use cases and start differentiating actions like that.